doc/howto/instances_routed_nic_vm.md

```diff
@@ -15,21 +15,20 @@ Then add the routed NIC device:
 
 In this command, `my-parent-network` is your parent network, and the IPv4 and IPv6 addresses are within the subnet of the parent.
 
-Next we will create a `netplan` configuration file `netplan.yaml` to use with the instance:
+Next we will add some `netplan` configuration to the instance using the `cloud-init.network-config` configuration key:
 
-    cat <<EOF > netplan.yaml
+    cat <<EOF | lxc config set jammy cloud-init.network-config -
     network:
       version: 2
       ethernets:
         enp5s0:
           routes:
           - to: default
             via: 169.254.0.1
+            on-link: true
           - to: default
             via: fe80::1
-          link-local:
-          - ipv4
-          - ipv6
+            on-link: true
```

Comment:
With the `on-link` changes, I think the following text snippet needs an update.

Reply:
@markylaing do you know what @simondeziel is asking for here?

Reply:
Ah yes of course!